examples/transitions/state_machine.act

```diff
@@ -1,39 +1,32 @@
 behaviour init of StateMachine
+interface constructor()
+
 creates
-	uint256 x := 0
+  uint256 x := 0
 
 invariants
-	x <= 2
-
+    x <= 2
 
 behaviour f of StateMachine
 interface f()
 
 case x == 0:
-	storage
-		x => 1
-
-case _:
```

Comment:
what does this machine do when `x /= 0`? This is currently unspecified. Should it be `iff x == 0` instead?

Reply:
If we are to follow the [code](https://github.com/ethereum/act/blob/coq/examples/transitions/state_machine.sol), we should probably keep the `case _: noop`  specified

Reply:
you're right, thanks for pointing that out. i'm not sure why i removed the catch-all case. patched in https://github.com/ethereum/act/commit/59aa11c9eb54ba1f41ef9b89b9b2508124d6c967. personally, i think it's cleaner to use an `iff`.

Reply:
hmm, but now it doesn't match the code. I'm fine with either, but if you want `iff` you need to change the code to use `requires` instead of `if`

Reply:
oops, right